CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## 5.0.5
+
+* Output a json log to stdout describing the result of the download
+  operation if the verbose flag is set to true.
```

Comment:
I think we should add a new flag for this rather than adding it to the existing verbose flag so that we don't significantly change the output for existing users that are running it with the verbose flag.

Reply:
I think a minor change to the output would be fine, but suddenly introducing a bunch of JSON that is sent to stdout is a pretty big change. The existing output is intended to be human-readable. This tool is used by tens of thousands of customers, many of whom have been using it for more than a decade. Introducing such a big change, would likely cause breakage and support requests, especially outside of a major version release.